src/gmail.js

```diff
@@ -2314,7 +2314,7 @@ var Gmail = function(localJQuery) {
     // console.log( "Observer set for", action, callback);
     api.observe.initialize_dom_observers = function() {
         api.tracker.dom_observer_init = true;
-        api.tracker.supported_observers = ["view_thread", "view_email", "load_email_menu", "recipient_change", "compose"];
+        api.tracker.supported_observers = ["view_thread", "view_email", "load_email_menu", "recipient_change", "recipient_change_new", "compose"];
```

Comment:
I don't think we wan't to add another event here. The best would be if existing code kept working unmodified by simply updateing `gmail.js`.

Do you think that could be possible?